staging/src/k8s.io/kubectl/pkg/cmd/create/create_cronjob.go

```diff
@@ -187,13 +189,17 @@ func (o *CreateCronJobOptions) Run() error {
 		if err != nil {
 			return fmt.Errorf("failed to create cronjob: %v", err)
 		}
+	} else {
```

Comment:
Can you just do this whole change in `createCronJob()` with no change needed in `Run()`?  I'm think this was what @soltysh meant (correct me if I'm wrong, @soltysh).

To see what I mean, look at an existing example from create_deployment.go, and it doesn't have an else block...  It just sets `meta.namespace` directly when it creates the &appsv1.Deployment...

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go#L153-L174

So for example, would something like this work?
```
func (o *CreateCronJobOptions) createCronJob() *batchv1beta1.CronJob {
        namespace := ""
	if o.EnforceNamespace {
		namespace = o.Namespace
	}
	return &batchv1beta1.CronJob{
		TypeMeta: metav1.TypeMeta{APIVersion: batchv1beta1.SchemeGroupVersion.String(), Kind: "CronJob"},
		ObjectMeta: metav1.ObjectMeta{
			Name: o.Name,
                        Namespace: namespace,
			},
		},
	}
}
```

Reply:
This part should be deleted, sorry I'll push code again.